src/main/java/io/arlas/server/wfs/utils/XmlUtils.java

```diff
@@ -31,6 +31,9 @@
 
 public class XmlUtils {
 
+
+    public static String charToUse;
```

Comment:
Could you find a more explicit name for this var? Maybe something mentioning "separator" or something like that. On top of that, related methods cleanName and retrieveOriginalName could also be more explicit.